consumer/src/main/kotlin/streams/StreamsEventSinkExtensionFactory.kt

```diff
@@ -81,6 +84,28 @@ class StreamsEventSinkExtensionFactory : KernelExtensionFactory<StreamsEventSink
         }
 
         private fun initSinkModule(streamsTopicService: StreamsTopicService, streamsSinkConfiguration: StreamsSinkConfiguration) {
+            if (streamsSinkConfiguration.checkApocTimeout > -1) {
+                GlobalScope.launch(Dispatchers.IO) {
+                    val success = StreamsUtils.blockUntilTrueOrTimeout(streamsSinkConfiguration.checkApocTimeout, streamsSinkConfiguration.checkApocInterval) {
+                        val hasApoc = Neo4jUtils.hasApoc(db)
+                        if (streamsLog.isDebugEnabled) {
```

Comment:
only if hasApoc is false.   Otherwise this will output the message every time, even after it succeeded.

Reply:
done, thanks for spotting this!